features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/recorder/mapper/TextViewMapper.kt

```diff
@@ -90,7 +90,7 @@ open class TextViewMapper<in T : TextView>(
         textAndInputPrivacy: TextAndInputPrivacy,
         isOption: Boolean
     ): String {
-        val originalText = textView.text?.toString().orEmpty()
+        val originalText = textView.layout?.text?.toString().orEmpty()
```

Comment:
should we fallback just to `textView.text` if `textView.layout` is `null` or it is not necessary?

Reply:
It can be `null` according to Javadoc: https://cs.android.com/android/platform/superproject/main/+/main:frameworks/base/core/java/android/widget/TextView.java;l=2629?q=TextView

Reply:
Nice catch, so it makes sense to fallback to the full text